src/3d-force-graph.js

```diff
@@ -204,9 +204,10 @@ export default Kapsule({
 			state.graphScene.add(node.__sphere = sphere);
 		});
 
-		const lineMaterial = new THREE.LineBasicMaterial({ color: 0xf0f0f0, transparent: true, opacity: state.lineOpacity });
 		state.graphData.links.forEach(link => {
 			const geometry = new THREE.BufferGeometry();
+            const color = link[state.colorField] || 0xf0f0f0;
+		    const lineMaterial = new THREE.LineBasicMaterial({ color: color, transparent: true, opacity: state.lineOpacity });
```

Comment:
@Jakanapes I like this feature, but I'm concerned about the performance impact of this change. 
Large data sets can have links in the order of tens of thousands and creating a dedicated line material for each will likely result in a performance degradation of the whole system.

Reply:
hmm, what if you created a line material for each unique link color, placed them in a map and assigned them that way?

So the most `new THREE.LineBasicMaterial` calls you would have would be n, where n is the number of unique line colors.

Reply:
Definitely! I've refactored the code a bit and merged.  Line color is now exposed as an accessor property via `linkColorField`. Available from `v1.10.0`.

Thanks for digging into this @Jakanapes!